cmd/image/qcow2ova/prep/templates_test.go

```diff
@@ -44,6 +44,12 @@ func TestRender(t *testing.T) {
 			"some-password",
 			false,
 		},
+		{
+			"rhel image without root password",
```

Comment:
prefer adding a test case where it rendered text shouldn't contain the `passwd root` text

Reply:
Done.